CHANGELOG.md

```diff
@@ -11,6 +11,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 #### Added
 
+- You can skip `--name` flag when using `account import` - a default name will be generated.
+
+## [Unreleased]
+
+### Cast
+
+#### Added
```

Comment:
You've duplicated the unreleased section

Reply:
Oopsie. I will remove it.

Reply:
There's another unreleased for a different feature added there. Seems to be different and not a duplicate

Reply:
There should be one `[Unreleased]` section. We change this into a relevant version when releasing.

Reply:
Got it. Thanks